File: plugin-api/src/main/java/api/CacheLoader.java
package api;

import org.displee.CacheLibrary;

public interface CacheLoader {

    void loadItemDefinitions(CacheLibrary library);

    void loadNpcDefintions(CacheLibrary library);

    void loadSequences(CacheLibrary library);

    void loadFrameArchive(int archiveId);
}

File: src/main/kotlin/cache/load/CacheLoader.kt
package cache.load

import org.displee.CacheLibrary

interface CacheLoader {

    fun loadSequences(library: CacheLibrary)

    fun loadFrameArchive(archiveId: Int)

    fun loadNpcDefintions(library: CacheLibrary)

    fun loadItemDefinitions(library: CacheLibrary)
}